src/lib/apiTypesV2/Attribute.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+#include <string>
+#include <vector>
+
+#include "common/tag.h"
+#include "apiTypesV2/Attribute.h"
+#include "ngsi10/QueryContextResponse.h"
+
+
+/* ****************************************************************************
+*
+* Attribute::render -
+*/
+std::string Attribute::render(ConnectionInfo* ciP, RequestType requestType, bool comma)
```

Comment:
v2 render() should be as "lean" as possible. Thus, I think that ciP and requestType should be removed (actually, I think they are not used in the function body). In addition, maybe the name of this function should be `toJson()` (not sure, but I think that APIv2 renders are using that name... check Ken's classes in apiTypesV2/).